meeting/14-12-18.md

```diff
@@ -0,0 +1,13 @@
+### Long Term goal:
+- Take more responsibility and manual test our code. (A friendly reminder)
```

Comment:
As in testing changes/new features for regressions/bugs more?

Reply:
Exactly 🥇

Reply:
I'm updating the reference notes now to include this :)

Reply:
thanks 🤛